docs/Classes.html

```diff
@@ -21,9 +21,9 @@
     <header class="header">
```

Comment:
do we really need `docs/` being checked in on the `master` branch? That'll make all diffs quite huge, right?

Reply:
Euhm, don't think so. I'll look into maybe having the docs on a separate branch or somewhere else entirely.

Reply:
docs branch is usually good, and you can self host the docs pages on gh pages then :) let us know if you need hints about is

Reply:
I'll look into updating this 👍

Reply:
I removed the docs for now. I'll add them back in on the /docs branch :)